src/Zone.js

```diff
@@ -10,15 +10,16 @@ import UXBase from './UXBase';
  * {
  *  name: "ux-destination-verified",
  *  marks: ["ux-image-online-logo", "ux-image-inline-logo"]
- *  onMeasure: measureName => {}
+ *  onMeasure: measureName => {},
  *  onMark: markName => {}
  * }
  */
 export default class Zone extends UXBase {
 	// Name used for UserTiming measures
 	measureName = this.props.name;
 
-	startMark = 'navigationStart';
+	// Name of the mark used a starting point of the measure
+	startMarkName = this.props.startMarkName;
```

Comment:
IMO we shouldn't alias this value - just use `this.props.startMarkName` in the `performance.measure` call directly. Just keeps the code more concise and the interface leaner.